script/migrations/index.sh

```
@@ -1,4 +1,4 @@
-#!/bin/bash
+﻿#!/bin/bash
```

Shellcheck (0.11.0)

[error] This file has a UTF-8 BOM. Remove it with: LC_CTYPE=C sed '1s/^...//' < yourscript .

(SC1082)